src/Graph.js

```diff
@@ -471,6 +471,11 @@ export const mkDemandSupply = function(board, options) {
 
 class DemandSupplyGraphAUC extends DemandSupplyGraph {
     drawAreaA(shadow=false, areaConf, intersection, l2) {
+        if (!l2) {
+            console.error('error: l2 is missing!');
```

Comment:
Are the `console.error`s needed?

Reply:
They're not needed, but they help with debugging if this problem occurs again.